core/server/models/user.js

```diff
@@ -38,11 +38,9 @@ User = ghostBookshelf.Model.extend({
     tableName: 'users',
 
     defaults: function defaults() {
-        var baseDefaults = ghostBookshelf.Model.prototype.defaults.call(this);
-
-        return _.merge({
+        return {
             password: security.identifier.uid(50)
-        }, baseDefaults);
```

Comment:
All of this weird logic to include the uuid is definitely not needed anymore though! Good spot 👀